src/runtime/particle-claim.ts

```diff
@@ -63,24 +66,27 @@ export class ClaimDerivesFrom {
   }
 
   toManifestString() {
-    return `derives from ${this.parentHandles.map(h => h.name).join(' and ')}`;
+    return `derives from ${this.parentHandles.map(h => h.name).join(' and derives from ')}`;
   }
 }
 
-export function createClaim(
+export function createParticleClaim(
     handle: HandleConnectionSpec,
     astNode: ParticleClaimStatement,
-    handleConnectionMap: Map<string, HandleConnectionSpec>): Claim {
-  let expression: ClaimExpression;
-  switch (astNode.expression.claimType) {
-    case ClaimType.IsTag:
-      expression = ClaimIsTag.fromASTNode(astNode.expression);
-      break;
-    case ClaimType.DerivesFrom:
-      expression = ClaimDerivesFrom.fromASTNode(astNode.expression, handleConnectionMap);
-      break;
-    default:
-      throw new Error('Unknown claim type.');
-  }
-  return new Claim(handle, expression);
+    handleConnectionMap: Map<string, HandleConnectionSpec>): ParticleClaim {
+  const claims: Claim[] = astNode.expression.map(claimNode => {
+  let claim: Claim;
+    switch (claimNode.claimType) {
+      case ClaimType.IsTag:
+        claim = ClaimIsTag.fromASTNode(claimNode);
```

Comment:
Just: `return ClaimIsTag.fromASTNode(claimNode);` No need for the `claim` variable

Reply:
Done. Thanks.